### PR TITLE
[10.0][T771]- Docker Build of Odoo

### PR DESCRIPTION
**Pull request by YogeshMahera-SerpentCS**
Task [771](https://www.quartile.co/web#id=938&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

**Comment by YogeshMahera-SerpentCS** [comment]
Closed PR #29 

Hello, @yostashiro @TimLai125 Amended your changes from closed #29  PR.

Can you please review this PR?

File: Dockerfile [hunk 1]
```diff
@@ -60,7 +62,7 @@ RUN chmod +x /odooboot
 # Get Odoo code
 WORKDIR /opt
 RUN set -x; \
-  git clone --depth 1 https://github.com/oca/ocb.git -b 10.0 odoo \
+  git clone --depth 1 https://github.com/odoo/odoo.git -b 10.0 odoo \
```
Comment (yostashiro) [inline Dockerfile:65]: Revert this change.

Comment (YogeshMahera-SerpentCS) [reply Dockerfile]: @yostashiro Changes Amended
